packages/dd-trace/test/appsec/rasp/command_injection.integration.spec.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const { createSandbox, FakeAgent, spawnProc } = require('../../../../../integration-tests/helpers')
+const getPort = require('get-port')
+const path = require('path')
+const Axios = require('axios')
+const { assert } = require('chai')
+
+// These test are here and not in the integration tests
+// because they require postgres instance
```

Comment:
It is not true for this test. It comes from the RASP SQLi integration test.

In the `integration-test` folder there are some RASP tests, so maybe this test should be moved there.

Reply:
copy paste failure, I should remove the comment.
I think that I like the idea to have the `integration.spec.js` tests here, I think that in a separate PR we should move ssrf integration test to this directory also. To have all the rasp tests (integration and not integration) in the same directory.